pkg/test/resourcefixture/testdata/basic/bigtable/v1beta1/bigtableappprofile/_http.log

```diff
@@ -6,17 +6,7 @@ GRPC /google.bigtable.admin.v2.BigtableInstanceAdmin/ListInstances
 
 OK
 
-{
-  "instances": [
```

Comment:
I'm taking a closer look at the diff for the `ListInstances` call. Was "data-boost-tf-test" created outside the test? I don't see it anywhere in the test file.  
Since this instance is removed in the mock log anyway, it's not a blocking comment.